docs/docs/contextual-conversations.mdx

```diff
@@ -1,120 +1,182 @@
+
+```rasa-yaml
+slots:
+  onboarding:
+    type: bool
+```
+
+### 3. Configuring the `TEDPolicy`
 
-Aside from the more rule-based policies we described above, Core also has some ML
-policies you can use. These come in as an additional layer in your policy configuration,
-and only jump in if the user follows a path that you have not anticipated. **It is important
-to understand that using these policies does not mean letting go of control over your
+In addition to adding stories to account for context,
+machine learning policies can help your model generalize
+to unseen conversation paths. **It is important
+to understand that using machine-learning policies does not mean letting go of control over your
 assistant.** If a rule based policy is able to make a prediction, that prediction will
-always have a higher priority (read more [here](./policies.mdx#action-selection)) and predict the next action. The
-ML based policies give your assistant the chance not to fail, whereas if they are not
-used your assistant will definitely fail, like in state machine based dialogue systems.
+always have a higher priority (read more [here](./policies.mdx#action-selection)) and predict the next action.
 
-These types of unexpected user behaviors are something our [TEDPolicy](https://blog.rasa.com/unpacking-the-ted-policy-in-rasa-open-source/) deals with
-very well. It can learn to bring the user back on track after some
-interjections during the main user goal the user is trying to complete. For example,
+The [`TEDPolicy`](https://blog.rasa.com/unpacking-the-ted-policy-in-rasa-open-source/) is made to handle
+unexpected user behaviors. For example,
 in the conversation below (extracted from a conversation on [Rasa X](https://rasa.com/docs/rasa-x/user-guide/review-conversations/)):
 
-```md
-## Story from conversation with a2baab6c83054bfaa8d598459c659d2a on November 28th 2019
-* greet
-  - action_greet_user
-  - slot{"shown_privacy":true}
-* ask_whoisit
-  - action_chitchat
-* ask_whatspossible
-  - action_chitchat
-* telljoke
-  - action_chitchat
-* how_to_get_started{"product":"x"}
-  - slot{"product":"x"}
-  - utter_explain_x
-  - utter_also_explain_nlucore
-* affirm
-  - utter_explain_nlu
-  - utter_explain_core
-  - utter_direct_to_step2
+```rasa-yaml
+stories:
+- story: Story from conversation with a2baab6c83054bfaa8d598459c659d2a on November 28th 2019
+  steps:
+  - checkpoint: STORY_START
+  - intent: greet
+  - action: action_greet_user
+  - slot_was_set:
+    - shown_privacy: true
+  - intent: ask_whoisit
+  - action: action_chitchat
+  - intent: ask_whatspossible
+  - action: action_chitchat
+  - intent: telljoke
+  - action: action_chitchat
+  - intent: how_to_get_started
+    entities:
+    - product: x
+  - slot_was_set:
+    - product: x
+  - action: utter_explain_x
+  - action: utter_also_explain_nlucore
+  - intent: affirm
+  - action: utter_explain_nlu
+  - action: utter_explain_core
+  - action: utter_direct_to_step2
```

Comment:
not sure how relevant this example is anymore tbh... let's leave it in for now, but i think we should in future consider having a whole example here that users can build from start to finish. maybe we can create an issue with follow up tasks?